sklearn/discriminant_analysis.py

```diff
@@ -537,7 +537,9 @@ def predict_log_proba(self, X):
         C : ndarray of shape (n_samples, n_classes)
             Estimated log probabilities.
         """
-        return np.log(self.predict_proba(X))
+        prediction = self.predict_proba(X)
```

Comment:
Thanks for the PR! This one might deserve a test, though I'm not entirely certain.

Reply:
This one is already tested. Many of the warnings where due to a np.log(0) that was happening in the tests